### PR TITLE
First Capstone Project- Conference Website

### PR DESCRIPTION
**Pull request by kelvinbush**
> In this pull request I did the following to set up, implement and deploy the `Crypto-Con Conference Website`

- I set up the html skeleton for the mobile and desktop versions of the app; the home and about page.
- I added fonts and image assets for the app.
- I added styling with sass.
- I added dynamic `Featured Speakers` with JavaScript.
- I set up mobile navigation with JavaScript.
- I set up linters for code quality.
- I deployed the website on this [link](http://kelvinbush.me/crypto-con/) using GitHub Pages.

Here is the video presentation [link](https://www.loom.com/share/3c42bf9084dd45da964029a631fdb6b7)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                        <img src="img/twitter.png" alt="Twitter"></a>
+                    <a href="https://www.facebook.com/CryptoCon2020/" target="_blank" rel="noopener noreferrer">
+                        <img src="img/facebook.png" alt="Facebook"></a>
+                    <a href="#"><p>English</p></a>
+                    <a href="#"><p>My Page</p></a>
+                    <a href="#"><p>Logout</p></a>
+                </div>
+            </div>
+            <nav>
+                <img id="toggle-nav" src="img/burger-bar.png" alt="Toggle Menu">
```
Comment (brytebee) [inline index.html:33]: - [ ] **Optional:** For a better UI, you could consider making the hamburger lighter when its background is dark. This is totally not required but my idea.

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            <img src="img/peercoin.svg" alt="Peer coin sponsor">
+            <img src="img/cardano.svg" alt="Tron sponsor">
+            <img src="img/solana-logo.svg" alt="Solana Sponsor">
+        </div>
+    </div>
+</div>
+<footer>
+    <img src="img/icons8-blockchain.png" alt="Logo">
+    <p>Copyright &copy; 2020 Crypto Con</p>
+</footer>
```
Comment (brytebee) [inline index.html:95]: - [ ] **Required:** You should space the footer according to the Behance template.

Your Design | Template
:---: | :---:
image | image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    <link rel="icon" href="img/icons8-blockchain.png" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,100;1,300&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet " href="css/style.css" >
+    <title>Crypto Con &mdash; Your Blockchain Conference</title>
+</head>
+<body>
+<div class="hero">
```
Comment (brytebee) [inline index.html:15]: - [ ] **Required:** This sections function appropriately. However, you are required to use more semantic tags throughout the website. Kindly replace most or all of the div tags in the project with more semantic tags.

image

File: sass/_components.scss [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+  &__email {
+    text-decoration: underline;
+    color: var(--color-primary);
+    font-weight: 700;
+    font-size: 1.4rem;
+  }
+}
+
+.logo {
+  display: flex;
```
Comment (brytebee) [inline sass/_components.scss:41]: - [ ] **Optional:** You have used the rule `display: flex;` multiple times, you could extract it into its own class and add it to your `HTML`. It's strongly suggested but not required.

File: sass/_components.scss [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+}
+
+.history {
+  background-color: #fff;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (brytebee) [inline sass/_components.scss:84]: - [ ] **Optional:** As mentioned above the rule `align-items: center;` could also be extracted into its class or made into one of your variables.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,722 @@
...
+  background: var(--color-secondary);
+}
+
+a {
+  text-decoration: none !important;
+}
+
+footer {
+  padding: 1.6rem;
+  display: flex;
```
Comment (brytebee) [inline css/style.css:45]: - [ ] **Optional:** It could be nice to extract some rule that re-occur like `display: flex;`

File: js/script.js [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+const programSection = document.getElementById('program');
+programSection.innerHTML = blockchainTopics.map((topic) => `
+  <div class="program__item">
+                  <img src="${topic.img}" alt="Program 1">
+                    <h3>${topic.topic}</h3>
+                    <p>${topic.description}</p>
+               </div>`)
+  .join('');
+
+const speakersList = [{
```
Comment (brytebee) [inline js/script.js:69]: 👏👏 Correctly handle an array of objects. Well done!

File: js/script.js [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+},
+{
+  topic: 'Ethereum',
+  description: 'Ethereum is a decentralized platform that is used to create blockchains.',
+  img: 'img/ethereum.png',
+},
+];
+
+const programSection = document.getElementById('program');
+programSection.innerHTML = blockchainTopics.map((topic) => `
```
Comment (brytebee) [inline js/script.js:61]: - [ ] **Required:** Kindly fix console errors as listed in the images below.

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @kelvinbush👋, 

Good job so far!

### Highlights 💡💡
- the website has been deployed.✔️
- the link to the live page has been correctly documented in the README.✔️
- the nav links are added and function as they should.✔️
- all pages contain the same navbar and footer.✔️
- design is mobile-first as required and media queries are used to adjust content to a desktop view. ✔️
- 👏👏 Correctly handle an array of objects. Well done!✔️
- Dynamic content is correctly attached to the static HTML.✔️
- no linter errors.✔️

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by keroloussamy — APPROVED** [review]
Hi @kelvinbush, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**